### PR TITLE
Realize calculator

### PR DESCRIPTION
**Pull request by NN171**
Added realization of calc's methods. Added tests for calc

### CODE REVIEW

File: src/main/java/Calculator.java [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-        //TODO inser your realisation in method div
+        return a / b;
     }
     public int times(int a, int b){
-        //TODO inser your realisation in method times
+        return a * b;
     }
     public int solver(){
-        //TODO inser your realisation in method solver
+        return times(dif(add(5, 3), 4), 6);
```
Comment (MacosTacos) [inline src/main/java/Calculator.java]: here you are not using div function

File: src/test/java/CalculatorTest.java [hunk 2]
```diff
@@ -1,31 +1,46 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.*;
```
Comment (MacosTacos) [inline src/test/java/CalculatorTest.java]: not used import

**Review by MacosTacos — CHANGES_REQUESTED** [review]
fix issuses please

File: src/test/java/CalculatorTest.java (hunk 2 above)
Comment (NN171) [reply src/test/java/CalculatorTest.java]: Removed unused imports

File: src/main/java/Calculator.java (hunk 1 above)
Comment (NN171) [reply src/main/java/Calculator.java]: Added

**Review by MacosTacos — APPROVED** [review]
good job brother

**Pull request merged**